test/e2e/__snapshots__/gymnast.spec.js.snap

```diff
@@ -3,7 +3,10 @@
 exports[`e2e tests should export the same values from the main export than individual files 1`] = `
 Object {
   "Col": [Function],
-  "ConfigProvider": [Function],
```

Comment:
with the update to not export config context, will we keep the same API?

Reply:
Once I make my changes, it will look like:
```
Object {
  "Col": [Function],
  "ConfigContext": Object {
    "Consumer": [Function],
  },
```

or another option to create a separate folder for `configConsumer`, then the snapshot could look like:
```
Object {
  "Col": [Function],
  "ConfigConsumer": [Function],
```

Reply:
why not keep the API the same? if we change it we'll have to have a separate v1 vs v0 docs

Reply:
Internally you see it as Provider / Consumer because we are implementing it using React Context API but the gymnast consumer should not be aware of these details.

Here we are exposing a ConfigProvider for the user to customize gymnast (a Component they can use to _provide_ configuration options).

It's a common pattern (both in naming and behavior) that [Redux](https://github.com/reduxjs/react-redux/blob/master/docs/api.md#provider-store) and [Apollo](https://www.apollographql.com/docs/react/essentials/get-started.html#creating-provider) use.

Reply:
Okay, the way Redux and Apollo are exposing their context is by externally exporting their `Provider` component while leaving their `Consumer` component un-exported, but providing a HOC component to access the context value (i.e. connect() in Redux and `<Query>` in Apollo).

So should your comment via `src/configContext/index.js` be reversed where we should externally export our `Provider` and not our `Consumer`?

Reply:
whichever one we export here should be the one exported as default on `configContext/index.js` or the lodash plugin will export the wrong thing